docs/UserGuide.md

```diff
@@ -318,7 +317,7 @@ Examples:
 Screenshot:
 
 ![Ui](images/ui-screenshots/friend-schedule.png)
-<br><center><ins>Image: gitGud after scheduling a friend's availability</ins></center>
+<br><center><ins>Image: Friend's schedule table updated after scheduling a friend's availability</ins></center>
```

Comment:
I feel this could potentially be a misleading screenshot (the user may think that after they run the command the main card will display the friend's info, perhaps can either add in the description that this is after friend --get, or not show it at all)

Reply:
> I feel this could potentially be a misleading screenshot (the user may think that after they run the command the main card will display the friend's info, perhaps can either add in the description that this is after `friend --get XXX`, or not show the main card at all)

Valid point, I think I will indicate that it is after running a 
```
friend --get XXX
```

Reply:
Resolved as discussed.